messaging/types/content_topic.go

```diff
@@ -0,0 +1,36 @@
+package types
+
+import (
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/status-im/status-go/eth-node/types"
+)
+
+const (
+	topicLength = 4
+)
+
+type ContentTopic [topicLength]byte
+
+func BytesToContentTopic(b []byte) (t ContentTopic) {
+	sz := topicLength
+	if x := len(b); x < topicLength {
+		sz = x
+	}
+	for i := 0; i < sz; i++ {
+		t[i] = b[i]
+	}
+	return t
+}
+
+func StringToContentTopic(s string) (t ContentTopic) {
+	str, _ := hexutil.Decode(s)
```

Comment:
What happens in case of a Decode error?

Reply:
`ContentTopic` will be empty then, that's _temporary_ copy paste from Waku's version.